### PR TITLE
new icon: playstation (original)

### PR DESCRIPTION
**Pull request by MichielPater**

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -2924,6 +2924,24 @@
         },
         "color": "#5058A6",
         "aliases": []
+    },
+	{
+        "name": "playstation",
+        "tags": [
+            "platform",
+            "cplusplus",
+			"game"
```
Comment (Panquesito7) [inline devicon.json:2933]:
```suggestion
            "platform",
            "cplusplus",
            "game"
```

File: devicon.json [hunk 2]
```diff
@@ -2924,6 +2924,24 @@
...
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#df0024",
+        "aliases": []
```
Comment (Panquesito7) [inline devicon.json:2944]:
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            }
        ]
```

File: icons/playstation/playstation-original.svg [hunk 3]
```diff
@@ -0,0 +1 @@
... height="128" fill-rule="evenodd" xmlns:v="https://vecta.io/nano"><path d="M68.332 35.996l-.082 74.27-19.828-6.473V16.5l25.313 6.715c16.203 4.371 26.117 12.863 25.957 27.75-.16 17.313-8.145 24.27-23.703 19.738V36.563c0-4.125-7.656-4.367-7.656-.566zm0 0" fill="rgb(87.058824%,0%,16.078431%)"/><path d="M39.309 89.391l-9.352 3.156c-6.047 2.105-11.203-2.832-5.641-4.855l4.512-1.617-17.008-5.422c-5.242 1.781-10.16 5.586-9.836 10.926.324 5.418 12.656 6.715 22.168 8.25 8.867 1.457 16.93.648 24.266-1.941v-5.582zm29.102 20.875l17.336-6.07-17.496-5.578v11.242zm0 0" fill="rgb(95.294118%,76.078431%,0.784314%)"/><path d="M114.523 94.004l.324-.082c7.578-2.668 10.801-6.391 9.996-9.867-1.289-5.824-10.562-8.98-24.832-10.035-10.234-.727-20.312 1.539-30.148 4.938l-1.613.566 17.656 5.5 10.32-3.48c10.801-2.02 15.156 1.539 4.754 4.855l-5.16 1.781zM48.422 68.277l-7.824 2.668 7.824 2.43zm0 0" fill="rgb(19.607843%,42.745098%,70.196078%)"/><path d="M85.746 104.195l28.777-10.191L95.82 88.18l-27.57 9.465v.973zM48.422 86.316l-9.113 3.074 9.113 2.914zm19.828 4.777v-11.57l17.656 5.5zm-39.422-5.02l19.59-7.035v-5.664l-7.82-2.43-28.375 9.629c-.082 0-.242.078-.402.078zm0 0" fill="rgb(0%,66.666667%,61.960784%)"/></svg>
```
Comment (Panquesito7) [inline icons/playstation/playstation-original.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128" fill-rule="evenodd" xmlns:v="https://vecta.io/nano"><path d="M68.332 35.996l-.082 74.27-19.828-6.473V16.5l25.313 6.715c16.203 4.371 26.117 12.863 25.957 27.75-.16 17.313-8.145 24.27-23.703 19.738V36.563c0-4.125-7.656-4.367-7.656-.566zm0 0" fill="rgb(87.058824%,0%,16.078431%)"/><path d="M39.309 89.391l-9.352 3.156c-6.047 2.105-11.203-2.832-5.641-4.855l4.512-1.617-17.008-5.422c-5.242 1.781-10.16 5.586-9.836 10.926.324 5.418 12.656 6.715 22.168 8.25 8.867 1.457 16.93.648 24.266-1.941v-5.582zm29.102 20.875l17.336-6.07-17.496-5.578v11.242zm0 0" fill="rgb(95.294118%,76.078431%,0.784314%)"/><path d="M114.523 94.004l.324-.082c7.578-2.668 10.801-6.391 9.996-9.867-1.289-5.824-10.562-8.98-24.832-10.035-10.234-.727-20.312 1.539-30.148 4.938l-1.613.566 17.656 5.5 10.32-3.48c10.801-2.02 15.156 1.539 4.754 4.855l-5.16 1.781zM48.422 68.277l-7.824 2.668 7.824 2.43zm0 0" fill="rgb(19.607843%,42.745098%,70.196078%)"/><path d="M85.746 104.195l28.777-10.191L95.82 88.18l-27.57 9.465v.973zM48.422 86.316l-9.113 3.074 9.113 2.914zm19.828 4.777v-11.57l17.656 5.5zm-39.422-5.02l19.59-7.035v-5.664l-7.82-2.43-28.375 9.629c-.082 0-.242.078-.402.078zm0 0" fill="rgb(0%,66.666667%,61.960784%)"/></svg>

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by amacado** [comment]
@Panquesito7 @Thomas-Boi and @everyone, does `playstation` fits into this iconset? Opinions please.

**Comment by Panquesito7** [comment]
> @Panquesito7 @Thomas-Boi and @everyone, does `playstation` fits into this iconset? Opinions please.

According to the icon tags, it says it's related to C++. If that's true, I have no problem with that, but still not sure if that fits our repository. 🤔

**Comment by Thomas-Boi** [comment]
Unfortunately, I think this doesn't fit the repo. Playstation here refers both to the company and the end-user hardware. It's not development centric enough. 

@MichielPater let us know why you think this fits the repo. Else, we'll have to close the PR

**Comment by MichielPater** [comment]
@Thomas-Boi @Panquesito7 @amacado I've developed for PlayStation using C++, so I see it as a target platform like Windows or Linux.

**Comment by amacado** [comment]
@MichielPater I understand your point but still not convinced adding playstation to this iconset. Maybe you can find some framework/sdk/... which is related to playstation development and have this added?

**Comment by MichielPater** [comment]
@amacado Yes, there is, but it's okay, thank you.

**Comment by Thomas-Boi** [comment]
@MichielPater I think the reason why we have Windows and Apple in the repo is because they are an essential part of the development process. While they are end-user products like Playstation, they also help programmers with tasks like running apps or GUI services. In cases like RaspberryPi, they provide a service that enables programmers to make mini-computers. 

For the Playstation, things are a bit different:
- It is mainly uses for gaming. While I'm sure it can be converted to a computer, its main use is not for that.
- Just being abled to create programs for it is not enough to count as a development tool. If we accept all hardware that a person can run programs on, we'll have to add things like the LG smart fridge or the Tesla electric car.

> Tools can be software, OS, services, etc. that helps with development. It must be specifically related to development (so software like Microsoft Word or Google Calendar won't be accepted since it's too general).

That being said, we are willing to accept the OS that the Playstation run. If I'm not mistaken, it's a Linux/Unix distro. Since a PS game developer will have to run the OS to test the app, that makes it a development tool.

**Pull request closed without merging**